docker-hub/billing/pricing-pull-rate-limit-faqs.md

```diff
@@ -0,0 +1,559 @@
+
+### Will I be rate-limited if I’m logged in but someone else on my IP hits the anonymous limit?
+
+No, users who are logged in and [authenticating their pull requests](https://docs.docker.com/docker-hub/download-rate-limit/#how-do-i-authenticate-pull-requests) will be limited based on their user account only. If an anonymous user on your IP is limited, you will not be affected as long as your requests are authenticated and you have not hit your own limit.
+
+### Does it matter what image I am pulling?
+
+No, all images are treated equally. The limits are entirely based on the account level of the user doing the pull, not the account level of the repository owner.
+
+### How will these limits be adjusted?
+
+We will be monitoring these limits closely and making sure they are appropriate for common use cases within their tier. In particular, free and anonymous limits should always support normal single-developer workflows. Adjustments will be made as needed in that spirit. You may also [submit input](https://github.com/docker/roadmap/projects/1) on the limits.
+
+### What about CI systems where pulls will be anonymous?
+
+We recognize there are some circumstances where many pulls will be made that can not be authenticated. For example, cloud CI providers may host builds based on PRs to open source projects. The project owners may be unable to securely use their project’s Docker Hub credentials to authenticate pulls in this scenario, and the scale of these providers would likely trigger the anonymous rate limits. We will unblock these scenarios as necessary and continue iterating on our rate limiting mechanisms to improve the experience, in cooperation with these providers. Please reach out to [support@docker.com](mailto:support@docker.com) if you are encountering issues.
+
+### Will Docker offer dedicated plans for open source projects?
+
+Yes, as part of Docker’s commitment to the open source community, we have created a new program to support free use of Docker in open source projects. [See this blog post](https://www.docker.com/blog/expanded-support-for-open-source-software-projects/) for more information.
+
+To apply for an open source plan, complete our application at:
+
+[https://www.docker.com/community/open-source/application](https://www.docker.com/community/open-source/application).
+
+For more information view our blogs on [inactive image retention](https://www.docker.com/blog/scaling-dockers-business-to-serve-millions-more-developers-storage) and [pull rate limits](https://www.docker.com/blog/scaling-docker-to-serve-millions-more-developers-network-egress).
+
+Or check out: [https://www.docker.com/pricing](https://www.docker.com/pricing)
+
+<script type="application/ld+json">
```

Comment:
@jhratliff Do we need this script in docs? Wondering if this got added by mistake?

Reply:
@usha-mandya - the script is what exposes the page content as structured data to the search crawlers. Based on the JSON data found on the page, the content will now be available in rich search features.

Reply:
Looks like the script is largely duplicating the whole page; wondering how we'll keep the script in sync with the actual page-content.

Wondering if there's a Jekyll plugin that allows us to annotate headings/sections to convert these to `ld+json`.

Or do you know if there's a way for this rich metadata to be constructed by Google / search engines from the actual page content (adding some markup/annotation to sections)?

Reply:
I tend to build these out as content fields in CMS templates so we only enter the data once in a DB structure. The template would then format the content for consumption by humans and crawlers. I haven't yet had a chance to dig into the Jekyll system to see what kind of data structures are available for this kind of use (I'm only seeing the static, hand-crafted content thus far).